core/modules/translation/translation.admin.inc

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * @file
+ * Administration form for node settings for content translation.
+ */
+
+ /**
+  * Builds the configuration form for node language configuration.
+  */
+function translation_node_configure_form() {
+
+  $form = array(
+    '#submit' => array('translation_node_configure_form_submit'),
+  );
+
+  $form['translation_show_translation_links'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Show content translation links'),
+    '#description' => t('Show the links to translations in content body and teasers. If you uncheck this option, switching language will only be available from the language switcher block.'),
+    '#default_value' => config_get('translation.settings', 'translation_show_translation_links'),
+    '#group' => 'translation_node_options',
+  );
+
+  $form['actions'] = array('#type' => 'actions');
+  $form['actions']['submit'] = array(
+    '#type' => 'submit',
+    '#value' => t('Save configuration'),
+  );
+
+  return $form;
```

Comment:
why not use the handy `system_settings_form`, to avoid having a custom submit handler ?

```
$form['#config'] = 'translation.settings';
return system_settings_form($form);
```